### PR TITLE
Feature/code updates

### PR DESCRIPTION
**Pull request by ZvonimirMedak**
## Summary

Updated the code style a bit, and added a part of the public documentation

**Related issue**: None

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [X] **Refactor**: This pull request refactors existing code.
- [X] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [X] This pull request introduces a **breaking change**.

### Description

Obj-c compatibility was removed since it was partially removed with the SwiftUI support.

Omitting a couple of returns, moving the code to new files/extensions, and adding some documentation.

## Checklist

- [X] I have performed a self-review of my own code.
- [X] I have tested my changes, including edge cases.
- [X] I have added necessary tests for the changes introduced (if applicable).
- [X] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

**Review by nikolamajcen — APPROVED** [review]
All good 👍

File: Sentinel/Classes/EmailSender/EmailSenderTool.swift [hunk 1]
```diff
@@ -79,17 +79,30 @@ public extension EmailSenderTool {
...
-            .padding()
+            ErrorView(alertTitle: alertTitle, alertMessage: alertMessage)
         }
     }
+
+}
+
+// MARK: - ErrorView
+
+private struct ErrorView: View {
```
Comment (nikolamajcen) [inline Sentinel/Classes/EmailSender/EmailSenderTool.swift:90]: I would recommend putting this into a new file and make it `internal` (no need to add that modifier explicitly), but to make it visible to internal implementation (e..g, if needed in other modules).

**Review by nikolamajcen — APPROVED** [review]
